graylog2-web-interface/src/components/event-definitions/event-definition-form/NotificationSettingsForm.jsx

```diff
@@ -19,6 +20,7 @@ class NotificationSettingsForm extends React.Component {
 
   state = {
     lastEnabledGracePeriod: undefined,
+    lastBacklogSize: undefined,
```

Comment:
With the current implementation we don't need to keep the backlog in the state, since it always have the same value as the props. This may be needed, though, if we want to use a `checkbox` to disable the backlog, as I suggested. That way we can use `0` internally but still display the last value the field had.